alarm/ExceptionManager.js

```diff
@@ -470,4 +471,64 @@ module.exports = class {
     }
   }
 
+  searchExceptionValue(exception, val2) {
```

Comment:
why not reuse the existing match logic? create a fake alarm object, and check if the exception matches.